### PR TITLE
Filter List Update (v7.0.0)

### PR DESCRIPTION
**Pull request by redtrillix**
# Description

[//]: # (Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.)

###### Filter List
- Update filter to v7.0.0.
- Changed email to support team email.
- Changed update time from AWST > UTC (Universal Coordinated Time).

Added filters for:
- www.techopedia.com
- office365atwork.com

###### Pull Request Template
Cleaned up template for pull requests:
- Added clearer comments for each section.
- Removed irrelevant sections with use with code.

## Type of change

[//]: # (Please delete options that are not relevant.)

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Filter-list change.

# Checklist:
[//]: # (Delete the ones that dont apply.)

- [ ] My changes follows the style guidelines of this project
- [x] I have performed a self-review of my changes
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: filters/filters.txt [hunk 1]
```diff
@@ -77925,3 +77925,12 @@ www.youtube.com###foreground-content
 ! youtube.com
 youtube.com###banner
 youtube.com##.style-scope.ytd-action-companion-ad-renderer.no-transition
+
+!!!! DRAFT !!!!
```
Comment (redtrillix) [inline filters/filters.txt:77929]: These need to be sorted in alphabetical order to match above filters, currently unable to so ill do it later.

File: .github/auto-assign.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  - redtrillix
+  
+# A number of reviewers added to the PR
+# Set 0 to add all the reviewers (default: 0)
+numberOfReviewers: 1
+
+# A list of assignees, overrides reviewers if set
+assignees:
+  - redtrillix
+
```
Comment (destronia-releases) [inline .github/auto-assign.yml:18]:
```suggestion
  
```

Comment (destronia-releases) [inline .github/auto-assign.yml:18]:
```suggestion

```

File: .github/auto-assign.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Set to true to add reviewers to PRs
+addReviewers: true
+
+# Set to 'author' to add PR's author as a assignee
+addAssignees: author
+
+# A list of reviewers to be added to PRs (GitHub user  name)
+reviewers:
+  - redtrillix
+
```
Comment (destronia-releases) [inline .github/auto-assign.yml:10]:
```suggestion

```

**Review by destronia-releases — APPROVED** [review]

**Pull request merged**